### PR TITLE
fix: Fix nav logo on smaller displays

### LINKED ISSUE #1007: Logo Should not be hidden on small screen
**Is your feature request related to a problem? Please describe.** The logo from the nav should not be hidden

WhatsApp Image 2020-10-30 at 21 15 56

### PR DESCRIPTION
**Pull request by redpfire**
This PR reveals the nav logo on smaller displays.

Closes #1007

### CODE REVIEW

**Review by dsnehasish74 — CHANGES_REQUESTED** [review]
Do one thing In the small screen where the hamburger  icon is coming . Shift this hamburger in the second nav ........ I mean for small screen make double nav . In first nav left the thing 
as it is and in the second nav put the hamburger icon then I guess it will solve the problem . For your under standing I am adding the screen shot 

Screenshot (366)

**Comment by redpfire** [comment]
ok. sorry for late response. working on it right away.

**Comment by redpfire** [comment]
@dsnehasish74 something like this?

image
image

File: assets/css/header.css [hunk 1]
```diff
@@ -186,7 +186,15 @@ a:active {
   }
 }
 
+.navbar.burger {
+  display: none;
+}
+
 @media (max-width: 425px) {
```
Comment (dsnehasish74) [inline assets/css/header.css:193]: Yah its great but can you make one more little change ? make this max-width 450px .

**Review by dsnehasish74 — CHANGES_REQUESTED** [review]
For bellow 380px the above nav is breaking its line can you make the logo and the text size or the margins between little smaller so that the line don't break.

File: assets/css/header.css (hunk 1 above)
Comment (redpfire) [reply assets/css/header.css:193]: sure thing

**Comment by redpfire** [comment]
@dsnehasish74 let me know if you like it now

**Review by dsnehasish74 — APPROVED** [review]
Yah it's looking really good now . @redpfire Thanks a lot for your beautiful work :)

**Comment by redpfire** [comment]
:)

**Comment by shahednasser** [comment]
Awesome thank you!

@all-contributors please add @redpfire for code and bug

**Pull request merged**